cmd/prometheus-kafka-consumer-group-exporter/kafka_consumer_group_exporter.go

```diff
@@ -81,7 +82,11 @@ func main() {
 		)
 		prometheus.DefaultRegisterer.MustRegister(collector)
 
-		log.Fatal(http.ListenAndServe(c.String("listen"), promhttp.Handler()))
+		l, err := net.Listen("tcp4", c.String("listen"))
```

Comment:
How about making the first parameter to `net.Listen(...)` a flag, too? That would allow someone to use this application with TCP6 if they'd like to.